js/src/Figure.ts

```diff
@@ -69,7 +69,7 @@ export class Figure extends widgets.DOMWidgetView {
         if (width_undefined && height_undefined) {
             // Same as the defaults in bqplot.less
             suggested_height = 480;
-            suggested_width = 640;
+            suggested_width = this.el.clientWidth;
```

Comment:
This does not work in render, since the DOM is not attached (it will give 0). I think we really should have some predefined width, in case the DOM is not attached (offscreen rendering).

Reply:
Yes I realised that later... That is why I've put the PR in WIP. I will revert this change.